tools/check_spelling_pedantic.py

```diff
@@ -17,6 +17,14 @@
 except NameError:
   pass
 
+try:
+  cmp
+except NameError:
+
```

Comment:
Nit: Prefer not to have blank line here.

Reply:
Me too but the __yapf__ tests fail without it.